tips-and-tricks.md

```diff
@@ -1,6 +1,56 @@
 # Tips and Tricks
 
-## Managing your data and the storage spaces
+## Python
+
+### Install miniconda
+
+This is the simplest approach if you are familiar with `conda`.
```

Comment:
Maybe, you can mention that Anaconda is already available via module but if you want your own installation you can follow this procedure.  For the former case, you can add this line to your `.bashrc` to have conda on each login:
```
module load anaconda-py3/2019.03
```

Reply:
I was thinking about mentioning `module` and linking to the AI doc for more details. In general I'd rather avoid having too many different solutions otherwise it very quickly become hard to figure out "OK but what is the recommended solution".

As an example, http://www.idris.fr/jean-zay/cpu/jean-zay-cpu-calculateurs-disques.html is very complete but it does not really answer the question: "OK but where do I put my 20TB dataset that I want to stay around?"

Reply:
> As an example, http://www.idris.fr/jean-zay/cpu/jean-zay-cpu-calculateurs-disques.html is very complete but it does not really answer the question: "OK but where do I put my 20TB dataset that I want to stay around?"

Yes, I'm totally agree with your example!

For this case maybe the simplest thing to do is just to add something like this to the title:
```
### Install miniconda (if you don't want to use the version of Anaconda provieded via module)